### PR TITLE
Add results page

### PR DESCRIPTION
**Pull request by tbarnahor**

### CODE REVIEW

**Review by karenav — APPROVED** [review]

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#content {
```
Comment (szwbar) [inline src/main/webapp/style.css]: I recommend giving explicit class names to the classes you're using, as you might have multiple lists or content classes. Same as you did with `.place-container`.

File: src/main/webapp/results.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<html>
+  <script src="script.js"></script>
+  <head>
+    <meta charset="UTF-8">
+    <title>capstone project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body onload="">
+    <div id="content">
+      <h1> Food Mood's recommendation for you:</h1>
```
Comment (szwbar) [inline src/main/webapp/results.html]: Redundant space?

File: src/main/webapp/results.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <ol id="place">
+        <div class="place-container">
+        <li> Foo sushi </li> <br>
```
Comment (szwbar) [inline src/main/webapp/results.html]: Another indent?

File: src/main/webapp/results.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body onload="">
+    <div id="content">
+      <h1> Food Mood's recommendation for you:</h1>
+      <ol id="place">
+        <div class="place-container">
+        <li> Foo sushi </li> <br> 
+          <a href="https:www.fusushi"> <span title = "Fu sushi">Restaurant's website</span> </a><br> 
+          <small> <a href="https:www.googlemapsfusushi"> <span title = "Fu sushi on google maps">Open in google maps</span> </a> </small>
```
Comment (szwbar) [inline src/main/webapp/results.html]: I think you can use actual links :D

**Review by szwbar — APPROVED** [review]
Some comments. 
Seems OK! I don't have any access to the rendered page though :( Can you share a link that shows me how it looks?

File: src/main/webapp/results.html (hunk 4 above)
Comment (tbarnahor) [reply src/main/webapp/results.html]: np.. I was lazy haha

**Pull request merged**